docs/tutorial/fdcurves.rst

```diff
@@ -47,17 +62,37 @@ The raw data can be accessed as well::
     force = fd.downsampled_force1y
     distance = fd.distance2
 
-    # Plot manually: FD curve
+Plot FD curve manually::
+
     plt.scatter(distance.data, force.data)
-    # Plot manually: force timetrace
-    plt.plot(force.timestamps, force.data)
+    plt.ylabel("Force (pN)")
+    plt.xlabel("Distance ($\mu$m)")
+    plt.title("Manually plotted fd curve")
+
+.. image:: figures/fdcurves/fdcurves_scatter_manual.png
+
+Plot force versus time manually::
+
+    plt.ylabel("Force (pN)")
+    plt.xlabel("Timestamps (ns)")
+    plt.title("Force vs Time")
+
+.. image:: figures/fdcurves/fdcurves_f_vs_time.png
 
 FD Ensembles
 ------------
 
-It's also possible to work with multiple FD curves as an ensemble::
+FD curves can be aligned by combining them in an fd ensemble.
+If all the fd curves of interest are in the same file, the ensemble can be defined as
+`fd_ensemble = lk.FdEnsemble(ensemble_file.fdcurves)`. If the fd curves are in different files, the ensemble can be defined as follows::
+
+    ensemble_file1 = lk.File("20210728-115145 FD Curve FD hairpin forward.h5")
+    ensemble_file1.fdcurves
 
-    fd_ensemble = lk.FdEnsemble(file.fdcurves)
+::
+
+    ensemble_file2 = lk.File('20210728-115145 FD Curve FD hairpin back.h5')
+    ensemble_file2.fdcurves
```

Comment:
I don't see the part where you merge them?

Reply:
Indeed forgot to add that part, updated it